src/System.Linq.Parallel/tests/Combinatorial/CancellationParallelQueryCombinationTests.cs

```diff
@@ -470,6 +470,16 @@ public static void ToList_OperationCanceledException_PreCanceled(LabeledOperatio
         }
 
         [Theory]
+        [MemberData(nameof(UnaryCancelingOperators))]
+        [MemberData(nameof(BinaryCancelingOperators))]
+        [MemberData(nameof(OrderCancelingOperators))]
+        public static void ToLookup_OperationCanceledException(Labeled<Func<CancellationToken, Operation>> source, Labeled<Func<Action, Operation>> operation)
+        {
+            Functions.AssertEventuallyCanceled((token, canceler) => Cancel(token, canceler, source, operation).ToLookup(x => x));
+            Functions.AssertEventuallyCanceled((token, canceler) => Cancel(token, canceler, source, operation).ToLookup(x => x, y => y));
```

Comment:
No, because then I don't have a guarantee it'll cancel:
`ToLookup` (and several of the other outputs, I should spot-check) doesn't check the token itself, it relies on the underlying enumerable doing so.  Since the cancellation check isn't performed for each iteration, once partitioning gets involved I have no guarantee it would be visible if the cancel action was called from the top level.

Err.. I think that's what it was.  I'm pretty sure I had test runs that didn't cancel because of this, but I've been sitting on this for a while.